Unable to create an ElasticSearch drain

When I try to create an ElasticSearch drain, I'll get this error while I'm providing credentials:

```
$ clever drain create --verbose --alias my-app ElasticSearch http://elasticsearchinstance-elasticsearch.services.clever-cloud.com --username john --password changeme
[DEBUG] Load configuration from /Users/john/.config/clever-cloud
[DEBUG] Loading app configuration from /Users/john/src/my-app/.clever.json
[DEBUG] Registering drain for app_fc5b6d60-2f66-11e8-8a3b-43a11d499aa2
[ERROR] Credentials are: optional for HTTP, mandatory for ElasticSearch and TCPSyslog/UDPSyslog don't need them.
```

--- a/spec/drain.spec.js
+++ b/spec/drain.spec.js
@@ -1,0 +1,19 @@
+'use strict';
+
+const expect = require('chai').expect;
+
+const drain = require('../src/models/drain.js');
+
+describe('drain', function () {
+
+  it('authorizes ElasticSearch with credentials', () => {
+    const isAuthorized = drain.authorizeDrainCreation('ElasticSearch', {username: 'john', password: 'changeme'});
+    expect(isAuthorized).to.be.true;
+  });
+
+  it('does not authorize ElasticSearch without credentials', () => {
+    const isAuthorized = drain.authorizeDrainCreation('ElasticSearch', {username: null, password: null});
+    expect(isAuthorized).to.be.false;
+  });
+
+});

--- a/src/models/drain.js
+++ b/src/models/drain.js
@@ -99,7 +99,7 @@ Drain.disable = function(api, appId, drainId) {
 
 Drain.authorizeDrainCreation = function(drainTargetType, drainTargetCredentials) {
     if (Drain.drainTypeExists(drainTargetType)) { // drain type exists   
-        var credentialsStatus = Drain.credentialsStatus(drainTargetType); // retrieve creds for drain type ('mandatory', 'optional', undefined)
+        var credentialsStatus = Drain.credentialsStatus(drainTargetType).credentials; // retrieve creds for drain type ('mandatory', 'optional', undefined)
         switch (credentialsStatus) {
             case CREDENTIALS.MANDATORY:
                 return Drain.credentialsExist(drainTargetCredentials);

